README-zh_CN.md

```diff
@@ -0,0 +1,15 @@
+# fgit
+一个在克隆项目时自动把 GitHub 链接替换为 fastgit 链接的工具。
+
+[English](./README.md)
+
+## 如何使用？
```

Comment:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines 
See more [here](https://github.com/updownpress/markdown-lint/blob/master/rules/022-blanks-around-headers.md).